tests/ZfcRbacTest/Factory/IsGrantedViewHelperFactoryTest.php

```diff
@@ -31,16 +31,15 @@ public function testFactory()
     {
         $serviceManager = new ServiceManager();
 
-        $pluginManager  = new HelperPluginManager();
-        $pluginManager->setServiceLocator($serviceManager);
+        $pluginManager  = new HelperPluginManager($serviceManager);
 
         $serviceManager->setService(
             'ZfcRbac\Service\AuthorizationService',
             $this->getMock('ZfcRbac\Service\AuthorizationServiceInterface')
         );
 
         $factory   = new IsGrantedViewHelperFactory();
-        $isGranted = $factory->createService($pluginManager);
+        $isGranted = $factory->createService($serviceManager);
```

Comment:
no, this is a change in zend service manager v3, the "main locator" instead of the plugin manager is passed